Godeps/Godeps.json

```diff
@@ -294,6 +294,16 @@
 			"Comment": "v1.14.12",
 			"Rev": "fde4ded7becdeae4d26bf1212916aabba79349b4"
 		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/private/protocol/eventstream",
```

Comment:
these are surprising additions... what pulled these in?

Reply:
It's pulled in by `kustomize -> go-getter -> aws`